zerver/tests/test_narrow.py

```diff
@@ -368,17 +426,21 @@ def test_successful_get_old_messages(self):
         narrow = [['pm-with', 'othello@zulip.com']]
         self.get_and_check_messages(dict(narrow=ujson.dumps(narrow)))
 
-        narrow = [dict(operator='pm-with', operand='othello@zulip.com')]
+        narrow = [dict(operator='pm-with', operand='othello@zulip.com')] # type: ignore
+        # mypy doesn't understand this type conversion as of 08/12/2016
         self.get_and_check_messages(dict(narrow=ujson.dumps(narrow)))
 
     def test_get_old_messages_with_narrow_pm_with(self):
+        # type: () -> None
         """
         A request for old messages with a narrow by pm-with only returns
         conversations with that user.
         """
         me = 'hamlet@zulip.com'
 
         def dr_emails(dr):
+            # type: (Any) -> List[text_type]
+            # TODO: Change this function to accept text_type and List[Dict[str, Any]]
```

Comment:
Yeah, I realised that was the case but wasn't sure how I should change it. Thanks!